contrib/craftscripts/maze.js

```diff
@@ -23,19 +23,19 @@ importPackage(Packages.com.sk89q.worldedit.blocks);
 usage = "<block> [width] [length] [height] [size] [thickness] flags\n";
 usage += "\n";
 usage += "Maze Flags:\n";
-usage += "• i adds an entry and an exit\n";
-usage += "• y places the entry and the exit randomly\n";
-usage += "• f adds a floor\n";
-usage += "• c adds a ceiling\n";
-usage += "• e places air blocks\n";
-usage += "• a places air blocks only\n";
-usage += "• v creates a vertical maze\n";
+usage += "� i adds an entry and an exit\n";
+usage += "� y places the entry and the exit randomly\n";
+usage += "� f adds a floor\n";
+usage += "� c adds a ceiling\n";
+usage += "� e places air blocks\n";
+usage += "� a places air blocks only\n";
+usage += "� v creates a vertical maze\n";
```

Comment:
I think you messed up the encoding on this file.
The encoding is supposed to be UTF-8, now it's Windows-1252